types/index.d.ts

```diff
@@ -480,7 +476,7 @@ declare namespace postgres {
     | boolean
     | Date // serialized as `string`
     | readonly JSONValue[]
-    | { toJSON(): any } // `toJSON` called by `JSON.stringify`; not typing the return type, typings is strict enough anyway
+    | { toJSON(): JSONValue } // `toJSON` called by `JSON.stringify`
```

Comment:
Types are already quite strict in this library, and enforcing types for `toJSON()` may cause troubles when integrating with others libraries that don't respect the `JSONValue` type definition. Anyway, it's not `postgres` responsibility to validate user-side `toJSON` implementations: users that care about proper types should type the `toJSON` definition instead.

```suggestion
    | { toJSON(): any } // `toJSON` called by `JSON.stringify`; not typing the return type, types definition is strict enough anyway
```

Reply:
Ok, I think that's fair 👍